### DISCUSSION TITLE
Forking Renovate Outage

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#19873] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Forking Renovate last ran successfully on [any of our repositories](https://github.com/ScribeMD) at 13:00 UTC on January 16th, 2023. Since then, checking the box to force a run in any of our dependency dashboards has had no effect. I suspect this may be an ops issue since there have been no releases or modifications to any of our Renovate configurations since Forking Renovate was last observed alive.

The following warning, which appears on all of our repositories, predates the outage:

```WARN: RepoCacheS3.getCacheFolder() - appending missing trailing slash to pathname```

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
Thanks for reporting, we'll look into it.

Converted to am app support discussion, assuming it's an Ops issue.

**Comment by rarkins** [comment]
Can you log into https://developer.mend.io and check what log info you're seeing on any of those repos? e.g. are there recent jobs showing up, do they have any logs available, etc?

**Reply by Kurt-von-Laven** [reply]
That's awesome! So stoked about the new Forking Renovate dashboard.

Each repository has a single pending job at the top with the following details:

```
Status: Pending
Engine: Renovate
Reason: Scheduled

No log is available for this job
Something went wrong and the log is not available
```

A number of completed jobs follow.

**Reply by rarkins** [reply]
Thanks for checking that. We'll hopefully figure out what's gone wrong and unblock it within the next few hours or half a day. The dashboard is a work in progress, but functionally complete. For example you should be able to click to view the job logs of the completed jobs below it. We'll "publicize" it once it's more polished but you're welcome to start using it once we get this blocked job sorted out.

**Comment by nabeelsaabna** [comment, marked as answer]
Hi @Kurt-von-Laven, the issue is fixed now.
Thanks for reporting

**Reply by Kurt-von-Laven** [reply]
The same thing is happening again; it started over the weekend.

**Reply by rarkins** [reply]
Could you describe exactly what you're seeing or not seeing this time?

**Reply by Kurt-von-Laven** [reply]
The issue is identical to the original (excepting the date at which it started of course).

> checking the box to force a run in any of our dependency dashboards has had no effect

**Reply by rarkins** [reply]
Our backend indicates that jobs have been running on repos in your org from the weekend and until now. In other words no signs of an outage.

**Reply by Kurt-von-Laven** [reply]
I suppose it's possible that the issue is specific to ScribeMD/template-repo this time? I checked the box on ScribeMD/template-repo#16 eighteen hours ago. Still getting nothing but 500s from the dashboard, otherwise I would investigate more myself.

**Reply by Kurt-von-Laven** [reply]
The outage resolved at 00:28 UTC on March 1st, 2023; thank you!
